.prettierrc

```
@@ -0,0 +1,15 @@
+{
+    "printWidth": 100,
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": false,
+    "quoteProps": "as-needed",
+    "jsxSingleQuote": false,
+    "trailingComma": "all",
+    "bracketSpacing": true,
+    "bracketSameLine": false,
+    "arrowParens": "always",
+    "proseWrap": "preserve",
+    "endOfLine": "lf"
+  }
```

É uma boa prática deixar uma linha em branco ao final de cada arquivo para evitar a exibição do ícone de `No newline at end of file` no GitHub.

```suggestion
  }

```